Pipfile

```diff
@@ -10,6 +10,9 @@ docutils = "==0.15"  # not a direct dependency, see https://github.com/pypa/pipe
 flake8 = "*"
 flake8-bugbear = "*"
 mypy = ">=0.812"
+types-dataclasses = ">=0.1.3"
```

Comment:
This should only be needed <3.7, can we express that here?

Reply:
Good call, will do that since IIRC it's possible.

Reply:
For the record, this is actually not possible due to `python_version=3.6` in `mypy.ini`.